README.md

```diff
@@ -167,6 +172,120 @@ Parameter | Description | Default
 `newImage` | StorageOS node container image to upgrade to |
 
 
+## Cleanup Old Configurations
+
+StorageOS creates and saves its files at `/var/lib/storageos` on the hosts. This
```

Comment:
We need to emphasise that this will remove all their data and it will be unrecoverable (big warning). We should also mention that in case of using external etcd, they should remove all the data in the storageos section. 
something like 
`ETCDCTL_API=3 /usr/local/bin/etcdctl --endpoints http://storageos-etcd-client:2379 del --prefix storageos`